packages/react-components/react-list-preview/package.json

```
@@ -1,8 +1,8 @@
 {
   "name": "@fluentui/react-list-preview",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

```suggestion
  "version": "0.0.0",
```

We don't need to bump manually